README.md

```diff
@@ -1 +1,16 @@
-# mobile-version-portfolio
\ No newline at end of file
+# MOBILE-VERSION-PORFOLIO
+* A mobile view of my portfolio project on [figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A27)
+# Built With
+* HTML 
+* CSS
+* Figma
+
+# Prerequisite
+* Modern Browser.Example: Firefox, chrome.
+* Windows 7 and above, MacOs, linux and android.
+# SET UP
+* Download the folder and click on index.html
+
+# AUTHOR
+👤 Aneal Laryea
+* Github: [@aneallaryea](https://github.com/aneallaryea100)
```

Comment:
- [ ] Kindly add the remaining headings of your README just as it is in the [Microverse README Template](https://github.com/microverseinc/readme-template). Check the link and please add the remaining headings and their texts after the Author heading you already have. It is always good to document your README as professionally as the template is given. 👍